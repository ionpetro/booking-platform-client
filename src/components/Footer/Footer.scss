.footer {
  height: 187px;
  width: 100%;
  background-color: $secondary;
  color: $white;
  display: flex;
  justify-content: space-between;
  padding: 63px 32px 56px 32px;

  &__main {
    &__img {
      @media #{$tablet} {
        width: 82px;
        height: 14px;
      }
    }

    &__copyright {
      @media #{$tablet} {
        font-size: 12px;
      }
    }
  }

  &__currency {
    margin-top: 32px;
    img {
      margin-left: 4px;
      filter: invert(1);
    }
    @media #{$tablet} {
      font-size: 12px;
      margin-top: 24px;
    }

    @media #{$tablet} {
      margin-top: 8px;
    }
  }

  @media #{$tablet} {
    height: 75px;
    padding: 18px 32px 18px 32px;
  }

  @media #{$mobile} {
    flex-direction: column;
    height: 98px;
    padding: 18px 32px 18px 32px;
  }
}
