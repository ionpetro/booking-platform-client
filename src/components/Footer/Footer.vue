<template>
  <div class="footer">
    <div class="footer__main">
      <!-- <img class="footer__main__img" src="@/assets/images/white.png" alt="White logo" /> -->
      <div class="body-small footer__main__copyright">
        Copyright Â© 2021 IonPetro. All rights reserved.
      </div>
    </div>
    <div class="footer__currency">
      Currency:<img src="@/assets/icons/Bitcoin.svg" alt="Bitcoin" />
      <strong> Bitcoin</strong>
    </div>
  </div>
</template>

<style src="./Footer.scss" lang="scss" scoped></style>
