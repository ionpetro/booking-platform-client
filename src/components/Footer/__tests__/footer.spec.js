import {mount} from '@vue/test-utils';
import Footer from '../Footer.vue';

describe('Footer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(Footer);
  });

  describe('when loaded', () => {
    it('renders correctly', () => {
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
