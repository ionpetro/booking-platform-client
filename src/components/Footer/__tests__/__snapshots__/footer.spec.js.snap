// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer when loaded renders correctly 1`] = `
<div
  class="footer"
>
  <div
    class="footer__main"
  >
    <img
      alt="White logo"
      class="footer__main__img"
      src="@/assets/images/white.png"
    />
     
    <div
      class="body-small footer__main__copyright"
    >
      
      Copyright Â© 2020 Blueground. All rights reserved.
    
    </div>
  </div>
   
  <div
    class="footer__currency"
  >
    
    Currency:
    <img
      alt="Bitcoin"
      src="@/assets/icons/Bitcoin.svg"
    />
     
    <strong>
       Bitcoin
    </strong>
  </div>
</div>
`;
